platform/security/idp/security-idp-server/src/test/groovy/org/codice/ddf/security/idp/server/IdpEndpointSpecTest.groovy

```diff
@@ -386,6 +400,56 @@ class IdpEndpointSpecTest extends Specification {
SI+PGRzOlRyYW5zZm9ybXM+PGRzOlRyYW5zZm9ybSBBbGdvcml0aG09Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvMDkveG1sZHNpZyNlbnZlbG9wZWQtc2lnbmF0dXJlIi8+PGRzOlRyYW5zZm9ybSBBbGdvcml0aG09Imh0dHA6Ly93d3cudzMub3JnLzIwMDEvMTAveG1sLWV4Yy1jMTRuIyIvPjwvZHM6VHJhbnNmb3Jtcz48ZHM6RGlnZXN0TWV0aG9kIEFsZ29yaXRobT0iaHR0cDovL3d3dy53My5vcmcvMjAwMC8wOS94bWxkc2lnI3NoYTEiLz48ZHM6RGlnZXN0VmFsdWU+WHFxMWQzS3E5L01jZ3NaSlRkY1dNdThLVGlRPTwvZHM6RGlnZXN0VmFsdWU+PC9kczpSZWZlcmVuY2U+PC9kczpTaWduZWRJbmZvPjxkczpTaWduYXR1cmVWYWx1ZT5pbU1xNk9HZHVYUmFnWW1kQVRzNEhzQy96bXljNHhUWjhsSHhxaFJsZWRzanM3R0Y2TDZ4RkhUR0ZCQ3BHODRtazR4cVRWU2thSEs1aVUySlh4VFBxUWpNd1NMbXlyeCtOS3BqbWc1Q2hnL0RUSFMvQS9YYTU5SnIyN1NOTGhTSGw0aWZrUEdvdDlHcE5BcEttUTBhYVRHaHlYUE5KN1JYb1doRTZhZXovWHdnbzBxMFI2SlQ0UWJGRVlFQUVjT0hxL2kxZFNWakNWQnN1TUlId0E1OHJFZHBFejI0MkhpYmpXVTFXL1lnZTU5dmlrK1NMUXp2RERFMDBGNnBweU5WZ0laVkFIWFhFODZndm5xZlQzYmJLK1dVV3NVbklBM2JJQVZtSU42aWhQbXdBb0YxOUVSdVVHbzd2aS9xUjlLSk9rWmRXRE5leUt6OVVCenRmWG5YZ2c9PTwvZHM6U2lnbmF0dXJlVmFsdWU+PGRzOktleUluZm8+PGRzOlg1MDlEYXRhPjxkczpYNTA5Q2VydGlmaWNhdGU+TUlJRFVqQ0NBcnVnQXdJQkFnSUlWQlpoVllBRk1pZ3dEUVlKS29aSWh2Y05BUUVGQlFBd2dZUXhDekFKQmdOVkJBWVRBbFZUTVFzdwpDUVlEVlFRSUV3SkJXakVNTUFvR0ExVUVDaE1EUkVSR01Rd3dDZ1lEVlFRTEV3TkVaWFl4R1RBWEJnTlZCQU1URUVSRVJpQkVaVzF2CklGSnZiM1FnUTBFeE1UQXZCZ2txaGtpRzl3MEJDUUVXSW1WdFlXbHNRV1JrY21WemN6MWtaR1p5YjI5MFkyRkFaWGhoYlhCc1pTNXYKY21jd0hoY05NVGd3TVRBMU1ESXhNekkxV2hjTk1Ua3dNVEExTURJeE16STFXakJLTVFzd0NRWURWUVFHRXdKVlV6RUxNQWtHQTFVRQpDQXdDUVZveEREQUtCZ05WQkFvTUEwUkVSakVNTUFvR0ExVUVDd3dEUkdWMk1SSXdFQVlEVlFRRERBbHNiMk5oYkdodmMzUXdnZ0VpCk1BMEdDU3FHU0liM0RRRUJBUVVBQTRJQkR3QXdnZ0VLQW9JQkFRQ3gyMmJnY1p0L0ZTMXh5S0JBS2lrRTVnTGlpemNFMVlWODBWdmEKaytEVnBPcUx4c2VidW9GL2lXakMyRWRJK1VCZkh6K2ZkTmxtbzJVa09MQ1R5VS90MGFSZFJkWnVQU28vM2ViUXRsRW15cEtKNjQ3dApIZVFzUEVPMS9Fc1lQZlBtRmVTWHZIUU5RZWxUYTlJWlFlMTErK0JoaERyTWNVTkNEMWRuajdrTE5GVkNuSzNwTGhqbjFCTCtweDNFCmt3bXFCZnBoSG0xVGo3V3pCSWY2Q2JCQ3B0WVNUY3FxbGRIelhzcXl5TnZTcVJxZWxkR1p2Vkhsb0xPTnd0OWJ6VFZrOHlibWIxRVAKajJqSU1VeFRPeGEwZFZVS0plSVozenlaNFZLekxmY2pVblhnbU5BcVgyVHl0eWpFODUwOVlFc1p2SExST3c0VkRWS1pVazUxcVZtQgpBZ01CQUFHamdZRXdmekFKQmdOVkhSTUVBakFBTUNjR0NXQ0dTQUdHK0VJQkRRUWFGaGhHVDFJZ1ZFVlRWRWxPUnlCUVZWSlFUMU5GCklFOU9URmt3SFFZRFZSME9CQllFRkV4U0hTZ2Zrbzl6UThyM0VKYXRNMFIyUUlZQ01COEdBMVVkSXdRWU1CYUFGT0ZVeDVmZkNzSy8KcVY5NFhqc0xLK1JJRjczR01Ba0dBMVVkRVFRQ01BQXdEUVlKS29aSWh2Y05BUUVGQlFBRGdZRUFkdzBtZ1BySVVVc1lKelRMSWN3bwo2TjdQY2p2YitOMUQvRjh3VHRtR2VjRWY3UG5zUm9RQkpJenViVDVYc1psWUFTVWhybE51TkFUSXpkSnkzc3hhbVY4OTBXcmxuVWZXClpIcVg3dnZFSWFnY205RXMvdTJiYTViU1h2NGZnd0VEdCtXdUhwQUpZSkxPZjN1aEE4ck9rZ0t1SVVVcDlYeG5waWw2TUNNeHBMcz08L2RzOlg1MDlDZXJ0aWZpY2F0ZT48L2RzOlg1MDlEYXRhPjwvZHM6S2V5SW5mbz48L2RzOlNpZ25hdHVyZT48c2FtbDI6TmFtZUlEIHhtbG5zOnNhbWwyPSJ1cm46b2FzaXM6bmFtZXM6dGM6U0FNTDoyLjA6YXNzZXJ0aW9uIiBGb3JtYXQ9InVybjpvYXNpczpuYW1lczp0YzpTQU1MOjIuMDpuYW1laWQtZm9ybWF0OnBlcnNpc3RlbnQiIE5hbWVRdWFsaWZpZXI9Imh0dHA6Ly9jeGYuYXBhY2hlLm9yZy9zdHMiPmFkbWluPC9zYW1sMjpOYW1lSUQ+PHNhbWwycDpTZXNzaW9uSW5kZXg+Nzk3ODwvc2FtbDJwOlNlc3Npb25JbmRleD48L3NhbWwycDpMb2dvdXRSZXF1ZXN0Pg=="
```

Comment:
❓ If possible, can we move these long certs to resource files so that they are easier to maintain / update?

Reply:
It's not a cert, it's a SAML request that spring is sending. It was the one that DDF couldn't read.